src/account-abstraction.md

```diff
@@ -0,0 +1,168 @@
+<h1>
+    Account Abstraction 
+</h1>
+
+
+Account Abstraction is a win for usability in web3 and on Starknet it has been supported from the beginning at the protocol level. For a smart contract to be considered an account contract it must at least implement the interface defined by SNIP-6. Additional methods might be required for advanced account functionality
+
+
+
+Account Abstraction (AA) replaces Externally Owned Accounts (EOA) with smart wallets, which are implemented as smart contracts. Each smart wallet contract must implement the SNIP-6 interface. This transformation allows for greater flexibility and customization compared to traditional EOAs.
+
+
+<b><h3> Enhancing Transaction Flexibility and Security with Account Abstraction </h3></b>
+
+Smart Contracts Paying for Transactions:    
+In the traditional model, users need EOAs to pay gas fees for executing smart contracts. With AA, smart contracts can be designed to handle these fees. This means that the logic within the contract can manage the transaction fees, allowing for more flexible and user-friendly applications. 
+    
+<br>Validation Abstraction:
+	Single Validation Method on Ethereum: On Ethereum Layer 1, transactions are validated using a single method, typically involving the signature of an EOA. This method verifies that the transaction was authorized by the account holder.
+
+Multiple Validation Methods in AA: 
+    AA allows for different types of validations. For instance, instead of just EOA signatures, a contract could accept other cryptographic signatures (like those from multi-signature wallets, hardware wallets, or even different cryptographic algorithms). This flexibility enables diverse and potentially more secure ways to validate transactions. 
```

Comment:
Change this to a bullet list
- Gas fee payment
- Validation abstraction
      - in ethereum
      - with AA

Reply:
> On Ethereum Layer 1, transactions are validated using a single method, typically involving the signature of an EOA

As there are only a single method, you can just say it uses the signature of the EOA. Not 'typically'.